README.md

```diff
@@ -4,7 +4,7 @@
 [![codecov](https://codecov.io/gh/notaryproject/notation-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/notaryproject/notation-go.svg)](https://pkg.go.dev/github.com/notaryproject/notation-go@main)
 
-A collection of libraries for supporting Notation sign, verify, push, pull of oci artifacts. Based on Notary V2 standard.
+A collection of libraries for supporting Notation sign, verify, push, pull of oci artifacts. Based on Notation standard.
```

Comment:
Notary Project

Reply:
If notation is also an implementation of Notary project specs, so if it is about following the specs, then suggest using Notary project, if it is pure notation, then `Notation` is more approporate.